rules/windows/process_creation/proc_creation_win_remote_access_tools_anydesk_set_password_via_cli.yml

```
@@ -0,0 +1,26 @@
+title: Remote Access Tool - Potential AnyDesk Set Password Via CLI
+id: b1377339-fda6-477a-b455-ac0923f9ec2c
+related:
+    - id: b1377339-fda6-477a-b455-ac0923f9ec2c
+      type: similar
+status: test
```

```suggestion
status: experimental
```
It is a new rule prone to FP with only the CommandLine.
The rule b52e84a3-029e-4529-b09b-71d19dd27e94 will match too

---

Done